setup.py

```diff
@@ -35,14 +35,14 @@
               'bin/ddg2pdb',
               'bin/ddg2summary'],
 
-    install_requires=[ 'biopython',
-                       'matplotlib',
-                       'numpy',
-                       'pandas',
-                       'logomaker',
-                       'scipy',
-                       'six',
-                       'openpyxl',
+    install_requires=[ 'biopython==1.84',
```

Comment:
why `==` and not `>=`? Feels like we are pinning a specific version for no real reason